mesheryctl/pkg/utils/fixtures/TestConfig.yaml

```diff
@@ -1,21 +1,21 @@
 contexts:
   gke:
     components:
-      - meshery-istio
+    - meshery-istio
     channel: stable
     endpoint: http://localhost:9081
     platform: gke
     token: default
-    version: v0.4.2
+    version: latest
   local:
     endpoint: http://localhost:9081
     token: default
     platform: docker
     components:
-      - meshery-istio
+    - meshery-istio
     channel: stable
     version: v0.5.10
```

Comment:
@asubedy, if we don't use a specific version number for each test of the stable release of an adapter, will this make sense in the [Compatibility Matrix](https://docs.meshery.io/installation/compatibility-matrix)? I suspect not.

Reply:
@asubedy?